### PR TITLE
Validates columns of model

### PR DESCRIPTION
**Pull request by Peterparcoeur**
Validates almost done. Let's check for the last one we need to `validates :start_date, :end_date, presence: true`, but i need to know which input i need to have to add manually start_date, end_date -> I have also created some users, review, game, reservation in our database (manually -_-)

### CODE REVIEW

File: app/models/reservation.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Reservation < ApplicationRecord
   belongs_to :user
   belongs_to :game
+
+  validates :total, presence: true
```
Comment (DumontNicolas) [inline app/models/reservation.rb]: Validates le status non? Et pour start_date end_date tu le fais pas parceque t'input pas les dates atm?

File: db/schema.rb [hunk 2]
```diff
@@ -43,10 +42,10 @@
     t.string "content"
     t.bigint "reservation_id", null: false
     t.bigint "user_id", null: false
-    t.string "role"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.integer "rating"
+    t.boolean "role"
```
Comment (DumontNicolas) [inline db/schema.rb:49]: C'est du code de cochon ça non?

Comment (Peterparcoeur) [reply db/schema.rb:49]: c'est du code de pgm

File: app/models/reservation.rb (hunk 1 above)
Comment (Peterparcoeur) [reply app/models/reservation.rb]: ouep my bad je peuc rajouter le status et j'ai update toutes les tables aussi donc la db normalement on touche plus

Comment (Peterparcoeur) [reply app/models/reservation.rb]: "atm" ?

**Pull request merged**